js/decklist/decklist.js

```diff
@@ -303,6 +307,56 @@ function sortDecklist(deck, sortorder) {
     for (i = 0; i < deck.length; i++) { deck[i] = [ deck[i][2], deck[i][0] ] }
   }
 
+  else if ( sortorder == 'type' ) {
+
+    var type_to_cards = {}
+
+    for (i = 0; i < deck.length; i++) {
+
+      // We're going to search by lower case
+      var lcard = deck[i][0].toLowerCase();
+
+      // Grab the card's type
+      if (lcard in cards) { type = cards[ lcard ]['t']; }
+      else { type = 'z'; } // Unknown
+
+      // Create the type subarray
+      if ( !(type in type_to_cards ) ) { type_to_cards[type] = []; }
+
+      // Fix the Aetherling issue until the PDF things supports it
+      lcard = lcard.replace('\u00c6', 'Ae').replace('\u00e6', 'ae');
+      deck[i][0] = deck[i][0].replace('\u00c6', 'Ae').replace('\u00e6', 'ae');
```

Comment:
We really should just make a function for this... Not a nitpick for this PR, but I may take this up separately.